frontend/src/metabase/ui/components/overlays/Tooltip/Tooltip.styled.tsx

```diff
@@ -18,8 +18,8 @@ export const getTooltipOverrides = (): MantineThemeOverride["components"] => ({
     },
     styles: theme => ({
       tooltip: {
-        backgroundColor: theme.fn.themeColor("bg-black"),
-        color: theme.white,
+        backgroundColor: "var(--mb-color-tooltip-background)",
+        color: "var(--mb-color-tooltip-text)",
```

Comment:
This applies to the regular Mantine tooltip, e.g. if you hover on the legend items.

<img src="https://github.com/user-attachments/assets/045761f8-c01d-42d2-a84f-04b1f3378503" width="300">